frontend/src/components/SignIn/SignIn.js

```diff
@@ -34,19 +35,19 @@ class SignIn extends Component {
       })
       .catch((error) => {
         if (error.message.includes('401')) {
-          const parentForm = document.getElementsByTagName('form')[1];
-          const errorDiv = document.createElement('p');
-          errorDiv.setAttribute('class', 'errorMessage');
-          errorDiv.innerHTML = 'Unable to log in. Check your email and password and try again.';
-          parentForm.insertAdjacentElement('beforeend', errorDiv);
+          this.setState({
+
+            validationErrorText: 'Please make sure you have the correct email and passwords!',
+
+          });
```

Comment:
I think the two blank lines are probably unnecessary, and makes reading the function call a little weird.